### PR TITLE
Nikita /  Phase 2 Implementing Design - Quick Start / Api Registration Page 

### PR DESCRIPTION
**Pull request by nikita-deriv**
Change quick start page and api registration page according to new design: https://redmine.deriv.cloud/issues/38644#Phase_2_Card_for_Design_-_Docs_page

### CODE REVIEW

**Review by yashim-deriv — APPROVED** [review]
LGTM

File: _includes/demos/balance-php.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        $msgPHP = json_decode($msg, 1);
+        /*
+         * Since there can be no guarantee of the order of the calls when using websocket we need
+         * to check that we have received a reply to the authorize call before sending the balance 
+         * request
+         */
+        if (isset($msgPHP["error"])) {
+            echo $msgPHP["error"]["message"];
+            $conn->close();
+        }else if ($msgPHP["msg_type"] == 'authorize') {
```
Comment (yashim-deriv) [inline _includes/demos/balance-php.php:35]:
```suggestion
        } else if ($msgPHP["msg_type"] == 'authorize') {
```

File: _includes/demos/balance-javascript.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+// You can get your token here https://app.deriv.com/account/api-token. 
+const token = ''; //Replace with your API token 
+
+ws.onopen = function (evt) {
+    ws.send(JSON.stringify({ "authorize": token })) // First send an authorize call.
+};
+
+ws.onmessage = function (msg) {
+    var data = JSON.parse(msg.data);
```
Comment (yashim-deriv) [inline _includes/demos/balance-javascript.js:26]:
```suggestion
    const data = JSON.parse(msg.data);
```

File: _includes/demos/buy-contract-javascript.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+// You can get your token here https://app.deriv.com/account/api-token. 
+const token = ''; //Replace with your API token 
+
+ws.onopen = function (evt) {
+    ws.send(JSON.stringify({ "authorize": token })) // First send an authorize call.
+};
+
+ws.onmessage = function (msg) {
+    var data = JSON.parse(msg.data);
```
Comment (yashim-deriv) [inline _includes/demos/buy-contract-javascript.js:30]:
```suggestion
    const data = JSON.parse(msg.data);
```

File: _includes/demos/buy-contract-javascript.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            "buy": 1,
+            "subscribe": 1,
+            "price": 10,
+            "parameters": { "amount": 10, "basis": "stake", "contract_type": "CALL", "currency": "USD", "duration": 1, "duration_unit": "m", "symbol": "R_10" }
+        }));
+    } else if (data.msg_type == 'buy') { //Our buy request was successful lets print the results. 
+        console.log("contract Id " + data.buy.contract_id + "\n");
+        console.log("Details " + data.buy.longcode + "\n");
+    } else if (data.msg_type == 'proposal_open_contract') { //Because we subscribed to the buy request we will receive updates on our open contract. 
+        var isSold = data.proposal_open_contract.is_sold;
```
Comment (yashim-deriv) [inline _includes/demos/buy-contract-javascript.js:50]:
```suggestion
        const isSold = data.proposal_open_contract.is_sold;
```

File: _includes/demos/proposal-javascript.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        "currency": "USD",
+        "duration": 1,
+        "duration_unit": "m",
+        "symbol": "R_100",
+        "barrier": "+0.1"
+    }));
+};
+
+ws.onmessage = function (msg) {
+    var data = JSON.parse(msg.data);
```
Comment (yashim-deriv) [inline _includes/demos/proposal-javascript.js:33]:
```suggestion
    const data = JSON.parse(msg.data);
```

File: _includes/demos/ticks-javascript.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+const WebSocket = require('ws');
+var ws = new WebSocket('wss://ws.binaryws.com/websockets/v3?app_id=1089');
+
+ws.onopen = function (evt) {
+    ws.send(JSON.stringify({ ticks: 'R_100' }));
+};
+
+ws.onmessage = function (msg) {
+    var data = JSON.parse(msg.data);
```
Comment (yashim-deriv) [inline _includes/demos/ticks-javascript.js:9]:
```suggestion
    const data = JSON.parse(msg.data);
```

**Review by yashim-deriv — APPROVED** [review]
The code samples are using the `var` keyword. I think we should use `const` or `let` for certain cases

**Review by yashim-deriv — APPROVED** [review]

**Comment by adamrrudolf** [comment]
close due to changes coming from https://github.com/deriv-pocs/deriv-developers-portal/pull/30/

**Pull request closed without merging**